tests/conftest.py

```diff
@@ -50,7 +50,9 @@ def get_test_file_json():
 
     def func(filename):
         file_path = _test_data_path / filename
-        return json.load(open(file_path, 'r'))
+        with open(file_path, 'r') as f:
```

Comment:
Ah right, I suppose we never closed the file, but this does it for us automatically. Nice catch!

Reply:
the warnings aka errors caught it 😏